inc/class.admin.php

```diff
@@ -383,11 +383,11 @@ public static function print_options( $option_data ) {
 						break;
 
 					case 'text':
-					default:
+					//default:
 						$input_type = '<input type="text" id="' . $option[0] . '" name="' . $option[0] . '" value="' . esc_attr( $option_actual[ $option[0] ] ) . '" class="' . $option[3] . '" />' . PHP_EOL;
 						break;
 
-					case 'number':
+					//case 'number':
```

Comment:
For backwards compatibility, you should probably not comment out `case 'number';`

Also, I am unclear what the purpose of changing the default input field type from text to number is, and whether this will create any issues of backwards compatibility.